modules/flowable-task-service/src/main/java/org/flowable/task/service/impl/HistoricTaskInstanceQueryImpl.java

```diff
@@ -1252,6 +1253,20 @@ public HistoricTaskInstanceQuery taskWithFormKey() {
         return this;
     }
 
+    @Override
+    public HistoricTaskInstanceQuery taskFormKey(String formKey) {
+        if(formKey == null){
```

Comment:
nit pick but the coding "standards" specify "if (" instead of "if(".  There are multiple occurrences in this PR.